app/helpers/hyrax/permission_levels_helper.rb

```diff
@@ -0,0 +1,30 @@
+module Hyrax
+  module PermissionLevelsHelper
+    ##
+    # @return [Hash<String, String>] a map from i18nized strings to permission
+    #   level keywords
+    def configured_permission_levels
```

Comment:
Can we get unit tests for the methods in this class?

Reply:
done!